widgetry/src/widgets/checkbox.rs

```diff
@@ -35,14 +35,14 @@ impl Checkbox {
     ) -> Widget {
         let label = label.into();
         let (off, hitbox) = Widget::row(vec![
-            GeomBatch::load_svg(ctx.prerender, "system/assets/tools/toggle_off.svg")
+            GeomBatch::load_svg(ctx.prerender, "system/assets/widgetry/toggle_off.svg")
```

Comment:
In an effort to ease eventual packaging of widgetry, I thought it made sense to keep widgetry assets separate from the game assets. Happy to revert this part if it's controversial.